.gitmodules

```diff
@@ -12,7 +12,7 @@
 	branch = support/HAFS
 [submodule "gfdl-tracker"]
 	path = sorc/hafs_vortextracker.fd
-	url = https://github.com/hafs-community/gfdl-tracker.git
+    url = https://github.com/BijuThomas-NOAA/gfdl-tracker.git
```

Comment:
Is there a reason why the .gitmodules file needs to be changed here?

Reply:
@evankalina, Thanks for bringing this to my attention. This is fixed.